bin/metrics-per-process.py

```diff
@@ -107,60 +109,115 @@ def find_pids_from_name(process_name):
             pass
   return pids
 
-def stats_per_pid(pid):
-  '''Gets process stats using psutil module
+def other_stats(process_handler, metrics_regexp):
```

Comment:
I think maybe something more like`additional_stats` is more descriptive.

Reply:
Done, let me know if I should squash all the commits

Reply:
Nah, its good github now has an easy squash + merge button.

Reply:
and honestly the history is not bad here.